dashboard/app/models/user.rb

```diff
@@ -48,6 +48,7 @@
 #  terms_of_service_version :integer
 #  urm                      :boolean
 #  races                    :string(255)
+#  primary_email_id         :integer
```

Comment:
nit: name the column as with the association below, `primary_authentication_option_id`

Reply:
Yup, this was a bug - I didn't migrate the database, fixed now.